addons/l10n_id_efaktur/models/account_move.py

```diff
@@ -1,71 +1,76 @@
 from odoo import api, fields, models, _
 from odoo.exceptions import UserError, ValidationError, RedirectWarning
 from odoo.tools import float_round, float_repr
 
-FK_HEAD_LIST = ['FK', 'KD_JENIS_TRANSAKSI', 'FG_PENGGANTI', 'NOMOR_FAKTUR', 'MASA_PAJAK', 'TAHUN_PAJAK', 'TANGGAL_FAKTUR', 'NPWP', 'NAMA', 'ALAMAT_LENGKAP', 'JUMLAH_DPP', 'JUMLAH_PPN', 'JUMLAH_PPNBM', 'ID_KETERANGAN_TAMBAHAN', 'FG_UANG_MUKA', 'UANG_MUKA_DPP', 'UANG_MUKA_PPN', 'UANG_MUKA_PPNBM', 'REFERENSI', 'KODE_DOKUMEN_PENDUKUNG']
-
-LT_HEAD_LIST = ['LT', 'NPWP', 'NAMA', 'JALAN', 'BLOK', 'NOMOR', 'RT', 'RW', 'KECAMATAN', 'KELURAHAN', 'KABUPATEN', 'PROPINSI', 'KODE_POS', 'NOMOR_TELEPON']
-
-OF_HEAD_LIST = ['OF', 'KODE_OBJEK', 'NAMA', 'HARGA_SATUAN', 'JUMLAH_BARANG', 'HARGA_TOTAL', 'DISKON', 'DPP', 'PPN', 'TARIF_PPNBM', 'PPNBM']
-
-
-def _csv_row(data, delimiter=',', quote='"'):
-    return quote + (quote + delimiter + quote).join([str(x).replace(quote, '\\' + quote) for x in data]) + quote + '\n'
-
 
 class AccountMove(models.Model):
     _inherit = "account.move"
 
     l10n_id_tax_number = fields.Char(string="Tax Number", copy=False)
     l10n_id_replace_invoice_id = fields.Many2one('account.move', string="Replace Invoice", domain="['|', '&', '&', ('state', '=', 'posted'), ('partner_id', '=', partner_id), ('reversal_move_ids', '!=', False), ('state', '=', 'cancel')]", copy=False, index='btree_not_null')
-    l10n_id_attachment_id = fields.Many2one('ir.attachment', readonly=True, copy=False)
-    l10n_id_csv_created = fields.Boolean('CSV Created', compute='_compute_csv_created', copy=False)
+    l10n_id_efaktur_document = fields.Many2one('l10n_id_efaktur.document', readonly=True, copy=False, string="e-Faktur Document")
     l10n_id_kode_transaksi = fields.Selection([
-            ('01', '01 Kepada Pihak yang Bukan Pemungut PPN (Customer Biasa)'),
-            ('02', '02 Kepada Pemungut Bendaharawan (Dinas Kepemerintahan)'),
-            ('03', '03 Kepada Pemungut Selain Bendaharawan (BUMN)'),
-            ('04', '04 DPP Nilai Lain (PPN 1%)'),
-            ('05', '05 Besaran Tertentu'),
-            ('06', '06 Penyerahan Lainnya (Turis Asing)'),
-            ('07', '07 Penyerahan yang PPN-nya Tidak Dipungut (Kawasan Ekonomi Khusus/ Batam)'),
-            ('08', '08 Penyerahan yang PPN-nya Dibebaskan (Impor Barang Tertentu)'),
-            ('09', '09 Penyerahan Aktiva ( Pasal 16D UU PPN )'),
-        ], string='Kode Transaksi', help='Dua digit pertama nomor pajak',
+            ('01', '01 To the Parties that is not VAT Collector (Regular Customers)'),
+            ('02', '02 To the Treasurer'),
+            ('03', '03 To other VAT Collectors other than the Treasurer'),
+            ('04', '04 Other Value of VAT Imposition Base'),
+            ('05', '05 Specified Amount (Article 9A Paragraph (1) VAT Law)'),
+            ('06', '06 Other Deliveries'),
+            ('07', '07 Deliveries that the VAT is not Collected'),
+            ('08', '08 Deliveries that the VAT is Exempted'),
```

Comment:
Looking at the upgrade PR, I came across this. I have trouble parsing them. Could the language be improved?

Reply:
Hello @cawo-odoo , this one here uses langauge from official translation so I'm afraid I can't change it unfortunately